src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.CompilationTracker.CompilationPair.cs

```diff
@@ -27,41 +27,6 @@ public CompilationPair(Compilation withoutGeneratedDocuments, Compilation withGe
 
                 public Compilation CompilationWithoutGeneratedDocuments { get; }
                 public Compilation CompilationWithGeneratedDocuments { get; }
-
-                public CompilationPair ReplaceSyntaxTree(SyntaxTree oldTree, SyntaxTree newTree)
```

Comment:
One of your other PRs made me realize we never moved more code onto this type, which would mean these methods come back. But I can git revert this file at that point.